west.yml

```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: f3f276dd2ecad981a2abf1d87c3d3d0b2e9d5373
+      revision: 76b0463a5a4b8b44cb4c82e5906711d18ae439db
```

Comment:
0a1854bb2c1eba17490d1c82ba1f749a5745fc1e
https://github.com/nrfconnect/sdk-zephyr/commit/0a1854bb2c1eba17490d1c82ba1f749a5745fc1e

Reply:
Updated, please merge.

Reply:
@jfischer-no  If you want me to change the hash yet again you just edit the comment I don't any notification unfortunately. Please ping me if you want me to a a change. 
I will update to: 7735bc6d171ab2fe6590ab47171f5decdbe2ddfd